spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configserver/pom.xml

```diff
@@ -66,10 +66,12 @@
 						<name>${env.IMAGE}</name>
 					</image>
 					<goal>build-image</goal>
-					<skip>${skip.build.image}</skip>
 				</configuration>
 				<executions>
 					<execution>
+						<configuration>
```

Comment:
this is the only change in this PR. I've realized that the place where we say "skip image building" is not correct. It does not impact us, but of course this clearly declares the intention.